crates/objc2-encode/src/encode.rs

```diff
@@ -712,6 +712,10 @@ encode_args_impl!(A, B, C, D, E, F, G, H, I);
 encode_args_impl!(A, B, C, D, E, F, G, H, I, J);
 encode_args_impl!(A, B, C, D, E, F, G, H, I, J, K);
 encode_args_impl!(A, B, C, D, E, F, G, H, I, J, K, L);
+encode_args_impl!(A, B, C, D, E, F, G, H, I, J, K, L, M);
+encode_args_impl!(A, B, C, D, E, F, G, H, I, J, K, L, M, N);
+encode_args_impl!(A, B, C, D, E, F, G, H, I, J, K, L, M, N, O);
+encode_args_impl!(A, B, C, D, E, F, G, H, I, J, K, L, M, N, O, P);
```

Comment:
Add changelog entry for this, and update the docs that say it is implemented for 12 arguments